pennylane/_grad.py

```diff
@@ -334,7 +334,8 @@ def _jacobian_function(*args, **kwargs):
                     "PennyLane has a new return shape specification that"
                     " may not work well with autograd and more than one measurement. That may"
                     " be the source of the error. \n\n"
-                    "See the documentation of qml.enable_return for more information."
+                    "See the documentation here for more information:\n"
+                    "https://docs.pennylane.ai/en/stable/introduction/returns.html"
```

Comment:
Is there a way to validate this is the correct link even though the page isn't up yet?

Reply:
Not that I can think of, but it also gives me worries! What I've done to test:
- Copy "introduction/returns.html"
- Add it on to the end of https://xanaduai-pennylane--4062.com.readthedocs.build/en/4062/
- Confirm :+1: